docs/developers/developer-reference/sc-to-sc-calls.md

```diff
@@ -0,0 +1,472 @@
+---
+id: sc-to-sc-calls
+title: SC to SC Calls
+---
+[comment]: # (mx-abstract)
+
+This guide provides an overview of the different types of smart contract calls that originate from other smart contract calls.
+
+[comment]: # (mx-context-auto)
+
+## Introduction
+
+Smart contract calls on MultiversX fall into two main categories: synchronous (`sync`) and asynchronous (`async`), each with distinct usage scenarios based on developer needs, and dApp architecture.
+
+
+[comment]: # (mx-context-auto)
+
+## Overview
+
+[comment]: # (mx-context-auto)
+
+### Sync calls vs. async calls
+
+
+A `sync` call is similar to regular function call in a program: it relies on a call stack, the current execution is paused, the call is executed immediately, and execution of the caller function resumes immediately after.
+
+An `async` call is similar to an asynchronous function call in a program, just like launching it on a different thread. The async call is not added to the same stack and does not interrupt the execution of the caller function.
+
+The main differences between the two are in this table: 
+
+| `sync` calls | `async` calls |
+| ------------ | ------------- |
+| Are executed immediately, inline. | Are executed after the current transaction is completed. |
+| Only work in the same shard. | Work both in-shard and cross-shard. |
+| Function results are available immediately. | Function results are only available later, in the callback, if a callback exists. |
+| A callee crash causes caller to immediately crash as well. | A callee crash does not cause caller to crash, the error can be caught in the callback, if it exists. |
+| Reentrancy issues possible. | Reentrancy not possible. |
+
+
+[comment]: # (mx-context-auto)
+
+### Transfer-execute calls
+
+Transfer-execute calls are basically async calls without callback. You can think of them as a "fire and forget" mechanism.
+
+They come in two flavors:
```

Comment:
it would be easier for the reader to see the flavors if they were highlighted

Reply:
We're using highlights for code only.
I can make it bold or italic.